### PR TITLE
SUS-4765 | GDPR GA/internal tracking changes + Featured Video autoplay after modal interaction

### PR DESCRIPTION
**Pull request by latata**
* load analytics.js immediately
* setup all global GA tracking functions immediately
* run GA `create` commands on tracking-opt-in callback (that queues GA events)
* queue internal tracking events
* setup Featured Video player on tracking-opt-in callback

@Wikia/sus @Wikia/adeng @Wikia/x-wing 
https://wikia-inc.atlassian.net/browse/SUS-4765
https://wikia-inc.atlassian.net/browse/SUS-4802
https://wikia-inc.atlassian.net/browse/SUS-4803
https://wikia-inc.atlassian.net/browse/SUS-4804

### CODE REVIEW

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js [hunk 1]
```diff
@@ -4,473 +4,473 @@
...
+						'userId': window.wgGAUserIdHash
+					}
+				);
+			}
 
-			args.unshift('send', 'event');
-			try {
-				_gaWikiaPush(args);
-			} catch (e) {
+			if (!optIn) {
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/js/universal_analytics.js:463]: 💎

**Review by nandy-andy — APPROVED** [review]

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js [hunk 2]
```diff
@@ -4,473 +4,473 @@
...
+				i[r].l = 1 * new Date();
+
+			// Insert the script tag asynchronously.  Inserts above current tag to prevent blocking in
+			// addition to using the async attribute.
+			a = s.createElement(o),
+				m = s.getElementsByTagName(o)[0];
+			a.async = 1;
+			a.src = g;
+			m.parentNode.insertBefore(a, m);
+		})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
```
Comment (mszabo-wikia) [inline extensions/wikia/AnalyticsEngine/js/universal_analytics.js:42]: is it OK to always load GA script itself?

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js [hunk 3]
```diff
@@ -4,473 +4,473 @@
...
+	} else {
+		// prevent errors when wgNoExternals is set or user opt-outs
```
Comment (mszabo-wikia) [inline extensions/wikia/AnalyticsEngine/js/universal_analytics.js]: comment needs an update

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js (hunk 2 above)
Comment (latata) [reply extensions/wikia/AnalyticsEngine/js/universal_analytics.js:42]: IMO it's fine as we always want to track all users opted-in/opted-out.

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**